docs/src/Run/Deployment.md

```diff
@@ -112,36 +117,86 @@ sudo docker swarm init --advertise-addr <manager-node-ip>
 On the manager node, get the join token:
 
 ```bash
-sudo docker swarm join-token worker
+>sudo docker swarm join-token worker
+
+docker swarm join --token SWMTKN-1-1tbk3g4qxoshrnzmx6a3fzoz9yyf6wxtaca33xwnt2fykd95et-1je480mao8ubve9xesiq3dym2 <manager-node-ip>:2377
 ```
 
+Save the join token for later use.
 On each worker node, run the join command provided, e.g.:
 
 ```bash
-sudo docker swarm join --token <token> <manager-node-ip>:2377
+sudo docker swarm join --token <token> <manager-node-ip>:2377 --advertise-addr <worker-node-ip>
+```
+
+Once you have your entire Swarm cluster set up, check the status on the manager node:
+
+```bash
+sudo docker node ls
+ID                            HOSTNAME           STATUS    AVAILABILITY   MANAGER STATUS   ENGINE VERSION
+q2lq4y0tzuwbrb17kddignc0p     ip-10-173-10-61    Ready     Active                          27.3.1
+6j201nmxjf54zwhjya0xxbl3d *   ip-10-173-10-112   Ready     Active         Leader           24.0.7
+ylett3pu2wz1p4heo1vdhz20w     ip-10-173-11-194   Ready     Active                          27.3.1
 ```
 
 ### 1.3 Deploying Gateway Services
 
 #### Step 1: Clone the Gateway Repository
 
-On the manager node:
-
 ```bash
 git clone https://github.com/ProjectLibertyLabs/gateway.git
-cd gateway
+cd gateway/deployment/swarm
 ```
 
 #### Step 2: Deploy the Stack
 
+The repo includes an example [docker-compose-swarm.yaml](../../../deployment/swarm/docker-compose.yaml) file for deploying the Gateway services on Docker Swarm.
+Edit the file to set the correct environment variables and service ports.
+Take note of the number of replicas for each service. The default is set to 3.
+
 ```bash
-sudo docker stack deploy -c docker-compose.yaml gateway
+sudo docker stack deploy -c docker-compose-swarm.yaml gateway
```

Comment:
It might be confusing to refer to this file as `docker-compose-swarm.yaml`, but the example file is `deployment/swarm/docker-compose.yaml`. I'd recommend one of the following:
* update the instructions in step 2 above to specifically copy & rename the example file
* Rename the example file to `deployment/swarm/docker-compose-swarm.yaml`
* Change the subsequent references to simpy refer to `docker-compose.yaml` to match the example filename (my least favoriate)